src/config/markets.ts

```diff
@@ -1,58 +1,129 @@
+    // OP [OP-USDC]
+    "0x4fDd333FF9cA409df583f306B6F5a7fFdE790739": DEFAULT_LISTING,
+    // PEPE [PEPE-USDC]
+    "0x2b477989A149B17073D9C9C82eC9cB03591e20c6": {
+      listingDate: p("17 Jul 2024"),
+    },
+    // WIF [WIF-USDC]
+    "0x0418643F94Ef14917f1345cE5C460C37dE463ef7": {
+      listingDate: p("17 Jul 2024"),
+    },
+    // SWAP-ONLY [USDC-USDC.e]
+    "0x9C2433dFD71096C435Be9465220BB2B189375eA7": DEFAULT_LISTING,
+    // SWAP-ONLY [USDC-USDT]
+    "0xB686BcB112660343E6d15BDb65297e110C8311c4": DEFAULT_LISTING,
+    // SWAP-ONLY [USDC-DAI]
+    "0xe2fEDb9e6139a182B98e7C2688ccFa3e9A53c665": DEFAULT_LISTING,
+  },
+  [AVALANCHE]: {
+    // BTC/USD [BTC-USDC]
+    "0xFb02132333A79C8B5Bd0b64E3AbccA5f7fAf2937": DEFAULT_LISTING,
+    // ETH/USD [ETH-USDC]
+    "0xB7e69749E3d2EDd90ea59A4932EFEa2D41E245d7": DEFAULT_LISTING,
+    // DOGE/USD [WAVAX-USDC]
+    "0x8970B527E84aA17a33d38b65e9a5Ab5817FC0027": DEFAULT_LISTING,
+    // SOL/USD [SOL-USDC]
+    "0xd2eFd1eA687CD78c41ac262B3Bc9B53889ff1F70": DEFAULT_LISTING,
+    // LTC/USD [WAVAX-USDC]
+    "0xA74586743249243D3b77335E15FE768bA8E1Ec5A": DEFAULT_LISTING,
+    // AVAX/USD [WAVAX-USDC]
+    "0x913C1F46b48b3eD35E7dc3Cf754d4ae8499F31CF": DEFAULT_LISTING,
+    // XRP/USD [WAVAX-USDC]
+    "0xD1cf931fa12783c1dd5AbB77a0706c27CF352f25": DEFAULT_LISTING,
+    // BTC/USD [BTC-BTC]
+    "0x3ce7BCDB37Bf587d1C17B930Fa0A7000A0648D12": DEFAULT_LISTING,
+    // ETH/USD [ETH-ETH]
+    "0x2A3Cf4ad7db715DF994393e4482D6f1e58a1b533": DEFAULT_LISTING,
+    // AVAX/USD [AVAX-AVAX]
+    "0x08b25A2a89036d298D6dB8A74ace9d1ce6Db15E5": DEFAULT_LISTING,
+    // SWAP-ONLY [USDC-USDT.e]
+    "0xf3652Eba45DC761e7ADd4091627d5Cda21F61613": DEFAULT_LISTING,
+    // SWAP-ONLY [USDC-USDC.e]
+    "0x297e71A931C5825867E8Fb937Ae5cda9891C2E99": DEFAULT_LISTING,
+    // SWAP-ONLY [USDT-USDT.e]
+    "0xA7b768d6a1f746fd5a513D440DF2970ff099B0fc": DEFAULT_LISTING,
+    // SWAP-ONLY [USDC-DAI.e]
+    "0xDf8c9BD26e7C1A331902758Eb013548B2D22ab3b": DEFAULT_LISTING,
+  },
+  [BSС_MAINNET]: {},
 };
 
-export const ENABLED_MARKETS_INDEX = mapValues<Record<number, string[]>>(ENABLED_MARKETS, (markets) =>
-  markets.reduce((acc, market) => ({ ...acc, [market]: true }), {})
+export const ENABLED_MARKETS_INDEX: Record<number, Record<string, true>> = mapValues(ENABLED_MARKETS, (markets) =>
+  mapValues(markets, () => true as const)
 );
 
 export function isMarketEnabled(chainId: number, marketAddress: string) {
   if (isDevelopment()) return true;
 
   return ENABLED_MARKETS_INDEX[chainId]?.[marketAddress] ?? false;
 }
+
+/**
+ * We let the APY to stabilize for a few days before showing it to the user
+ */
+export function getFutureDateForApyReadiness(chainId: number, marketAddress: string): undefined | Date {
+  const tokenListing = ENABLED_MARKETS[chainId]?.[marketAddress];
+
+  if (!tokenListing) return undefined;
+
+  const enoughDateForApy = addDays(tokenListing.listingDate, ENOUGH_DAYS_FOR_APY);
```

Comment:
not sure if we should move this logic to AprInfo or even separate utililty file?
cos `config/tokens` provides fairly straightforward getters/setters whereas here it clamps dates

Reply:
but its basically a getter

Reply:
i could move it tot aprInfo but i would need to pass token address to apy unfo. if you think that is ok let me know.